app/helpers/application_helper.rb

```diff
@@ -28,4 +28,25 @@ def markdown(text)
       Redcarpet::Markdown.new(renderer, extensions).render(text).html_safe
     end
   end
+
+  # Format create/update times on posts and comments.
+  def timestamp(item)
+    # TODO: make datetime formats configurable.
+    Time::DATE_FORMATS[:ddmmyyyy_hhmm] = '%d %B %Y %H:%M'
+    Time::DATE_FORMATS[:hhmm] = '%H:%M'
+
+    tz = '-08:00'   # Pacific time. TODO: make time zone configurable.
+    latest = item.updated_at.getlocal(tz).to_s(:ddmmyyyy_hhmm)
+    original = if (item.updated_at < item.created_at.end_of_day)
```

Comment:
Seems like this would always be true. Was one of the instances of `item.update_at` supposed to be `Time.zone.now`?

*NOTE:* `Time.zone.now` uses the request's default timezone, which can be set with `Time.zone =`. If you do that, you should be able to remove all the `getlocal` calls. I think that would be best done in a `before_action` in the `ApplicationController`.